templates/contribute.pug

```diff
@@ -12,6 +12,6 @@ block content
 
       p Several issues are raised across the hapijs organization that are waiting for contributors to show them some love. Those are marked with <a href="https://github.com/search?q=is%3Aissue+user%3Ahapijs+label%3A%22backlog%22"> backlog label</a>.
 
-      p We also labelled issues for newcomers that would like to contribute to our community. They are marked with <a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Ahapijs+label%3A%2hgood+first+issue%22+">good first issue label</a>.
+      p We also labelled issues for newcomers that would like to contribute to our community. They are marked with <a href="https://github.com/hapijs/hapi/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22">good first issue label</a>.
```

Comment:
The intent of the link is not to search for issues labelled with "good first issue" inside the `hapi` project itself but rather search for issues amongst the whole hapijs org.
If you want to update the link you have to put this URL instead: https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22good+first+issue%22+user%3Ahapijs+is%3Aopen